src/EditorFeatures/CSharpTest/QuickInfo/SyntacticQuickInfoSourceTests.cs

```diff
@@ -291,7 +292,8 @@ private QuickInfoProvider CreateProvider(TestWorkspace workspace)
             Assert.NotEqual(0, info.RelatedSpans.Length);
 
             var trackingSpan = new Mock<ITrackingSpan>(MockBehavior.Strict);
-            var quickInfoItem = await IntellisenseQuickInfoBuilder.BuildItemAsync(trackingSpan.Object, info, snapshot, document, CancellationToken.None);
+            var streamingPresenters = workspace.ExportProvider.GetExports<IStreamingFindUsagesPresenter>();
+            var quickInfoItem = await IntellisenseQuickInfoBuilder.BuildItemAsync(trackingSpan.Object, info, snapshot, document, streamingPresenters, CancellationToken.None);
```

Comment:
fascinating.  What's the reason QI would now have a dep on the FindRefs presenter?

Reply:
`cref` items are hyperlinked now. If you click on a type with multiple definitions (e.g. a partial type with multiple parts), it does a Find All References instead of Navigate To.